.yamato/training-int-tests.yml

```diff
@@ -43,6 +40,59 @@ test_mac_training_int_{{ editor.version }}:
         - "artifacts/inference.onnx.txt"
     standalonebuild:
       paths:
-        - "artifacts/testplayer*/**"
+        - "artifacts/testPlayer*/**"
         - "artifacts/models/**"
 {% endfor %}
+
+
+test_mac_backcompat_2020.1:
```

Comment:
Will move this to its own file with a nightly trigger instead of running on every PR.